recipes-core/base-files/base-files/system-backup.txt

```diff
@@ -5,6 +5,7 @@
 /etc/group
 /etc/gshadow
 /etc/sudoers.d
+/etc/hostname
```

Comment:
I'm fine with backing up `/etc/hostname`, though I wonder if it would make more sense to add it to `base-files`'s conffiles. Might be even worth to send upstream.

Also the installer [sets up `/etc/hosts` to contain an entry for the hostname](https://github.com/bisdn/meta-switch/blob/main/scripts/installer/install.sh#L454-L460), so you should back that up as well.

Reply:
Well, if we are looking at files other than /etc/hostname, I have a few more. PR updated.

Reply:
`/etc/hosts` is directly related to `/etc/hostname`, since the hostname entry there should match `/etc/hostname`. The ssh host keys are unreleated to the hostname.